avatars/avatars.js

```diff
@@ -3062,13 +1517,12 @@ class Avatar {
         }
       }
     };
-    _updateEyeTarget();
 
     const _updateEyeballTarget = () => {
       const leftEye = this.modelBoneOutputs['Eye_L'];
       const rightEye = this.modelBoneOutputs['Eye_R'];
 
-      const lookerEyeballTarget = this.looker.update(now);
+      const lookerEyeballTarget = this.looker.update(now, this);
```

Comment:
If `this` is needed every frame, why not pass it in the constructor?

Reply:
that does make more sense, will adjust